Setup/Uninstall.php

```diff
@@ -53,7 +53,7 @@ public function uninstall(SchemaSetupInterface $setup, ModuleContextInterface $c
         }
         //remove cron entries
         $collection = $this->collectionFactory->create();
-        $collection->addFieldToFilter('path', 'crontab/default/jobs/doofinder_update_on_save/schedule/cron_expr');
+        $collection->addFieldToFilter('path', 'doofinder_config_config/update_on_save/cron_expression');
```

Comment:
Is this key "doofinder_config_config" intended? why there are 2 config?

Reply:
For some reason we were using two keys differents for the same approach, I've been doing some tests and seems that the valid is the defined here
![image](https://github.com/user-attachments/assets/1115ccba-df70-4903-9b34-eef86204bc02)

so the other was inserted in the database but never used, however when uninstalling we were deleting the cronjob for the non-working but keeping the correct one.. so to avoid confusing I've keept the one that seems to be working and removed the other.